<ui version="4.0" >
 <class>OgrConverterGuiBase</class>
 <widget class="QDialog" name="OgrConverterGuiBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>OGR Layer Converter</string>
  </property>
  <property name="windowIcon" >
   <iconset>
    <normaloff/>
   </iconset>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="srcGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Source</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="labelSrcFormat" >
        <property name="minimumSize" >
         <size>
          <width>52</width>
          <height>22</height>
         </size>
        </property>
        <property name="text" >
         <string>Format</string>
        </property>
        <property name="buddy" >
         <cstring>comboSrcFormats</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="5" >
       <widget class="QComboBox" name="comboSrcFormats" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
        <property name="maxCount" >
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2" >
       <widget class="QRadioButton" name="radioSrcFile" >
        <property name="text" >
         <string>File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QRadioButton" name="radioSrcDirectory" >
        <property name="text" >
         <string>Directory</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" colspan="2" >
       <widget class="QRadioButton" name="radioSrcProtocol" >
        <property name="text" >
         <string>Remote source</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelSrcDataset" >
        <property name="text" >
         <string>Dataset</string>
        </property>
        <property name="buddy" >
         <cstring>inputSrcDataset</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4" >
       <widget class="QLineEdit" name="inputSrcDataset" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="5" >
       <widget class="QPushButton" name="buttonSelectSrc" >
        <property name="text" >
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <widget class="QLabel" name="labelSrcLayer" >
        <property name="minimumSize" >
         <size>
          <width>52</width>
          <height>22</height>
         </size>
        </property>
        <property name="text" >
         <string>Layer</string>
        </property>
        <property name="buddy" >
         <cstring>comboSrcLayer</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5" >
       <widget class="QComboBox" name="comboSrcLayer" >
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="dstGroupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Target</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QLabel" name="labelDstFormat" >
        <property name="minimumSize" >
         <size>
          <width>52</width>
          <height>22</height>
         </size>
        </property>
        <property name="text" >
         <string>Format</string>
        </property>
        <property name="buddy" >
         <cstring>comboDstFormats</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QComboBox" name="comboDstFormats" >
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
        <property name="maxCount" >
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="labelDstDataset" >
        <property name="text" >
         <string>Dataset</string>
        </property>
        <property name="buddy" >
         <cstring>inputDstDataset</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="inputDstDataset" >
        <property name="minimumSize" >
         <size>
          <width>200</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="buttonSelectDst" >
        <property name="text" >
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QLabel" name="labelDstLayer" >
        <property name="minimumSize" >
         <size>
          <width>52</width>
          <height>0</height>
         </size>
        </property>
        <property name="text" >
         <string>Layer</string>
        </property>
        <property name="buddy" >
         <cstring>inputDstLayer</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2" >
       <widget class="QLineEdit" name="inputDstLayer" >
        <property name="minimumSize" >
         <size>
          <width>300</width>
          <height>25</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <tabstops>
  <tabstop>comboSrcFormats</tabstop>
  <tabstop>radioSrcFile</tabstop>
  <tabstop>radioSrcDirectory</tabstop>
  <tabstop>radioSrcProtocol</tabstop>
  <tabstop>inputSrcDataset</tabstop>
  <tabstop>buttonSelectSrc</tabstop>
  <tabstop>comboSrcLayer</tabstop>
  <tabstop>comboDstFormats</tabstop>
  <tabstop>inputDstDataset</tabstop>
  <tabstop>buttonSelectDst</tabstop>
  <tabstop>inputDstLayer</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>OgrConverterGuiBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>424</x>
     <y>322</y>
    </hint>
    <hint type="destinationlabel" >
     <x>481</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>OgrConverterGuiBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>365</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel" >
     <x>281</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
